filament/src/details/View.h

```diff
@@ -205,6 +206,14 @@ class FView : public View {
         return mToneMapping;
     }
 
+    void setColorGrading(FColorGrading* colorGrading) noexcept {
+        mColorGrading = colorGrading;
```

Comment:
should we make it that if passing `nullptr` this reverts to the default one?

Reply:
Yes, good catch.